platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/workflow/task/delegation/WorkflowTaskDelegationSaveHook.java

```diff
@@ -0,0 +1,19 @@
+package com.softicar.platform.workflow.module.workflow.task.delegation;
+
+import com.softicar.platform.emf.table.listener.IEmfSaveHook;
+import com.softicar.platform.workflow.module.workflow.task.WorkflowTaskNotificationSubmitter;
+import java.util.Collection;
+
+public class WorkflowTaskDelegationSaveHook implements IEmfSaveHook<AGWorkflowTaskDelegation> {
```

Comment:
I think this needs unit test coverage.

Reply:
I dont think so.
The saveHooks themselves should already have unit tests, and all im doing here is using an already tested class.
As such, I dont really know what I should test here specifically

Reply:
But I added another test case in WorkflowTaskNotificationSubmitterTest

Reply:
I think that's exactly what should be tested.